samples/js/mocha/build.gradle

```diff
@@ -36,9 +36,9 @@ dependencies {
     testImplementation "org.jetbrains.kotlin:kotlin-test-js"
 }
 
-// we only configure compileTestKotlin2Js and not compileKotlin2Js as well
-// so that you can also use UMD as moduleKind for compileKotlin2Js
-compileTestKotlin2Js.configure {
+// we only configure compileTestKotlinJs and not compileKotlinJs as well
+// so that you can also use UMD as moduleKind for compileKotlinJs
+compileTestKotlinJs.configure {
     // mocha requires commonjs
     kotlinOptions.moduleKind = "commonjs"
```

Comment:
Do you know if we still need line 49 and 50 with the new plugin or is this now on per default?

Reply:
You are correct that the source maps are generated by default now, I will remove these lines.